docs/managing-dragonfly/tiering.md

```diff
@@ -0,0 +1,72 @@
+---
+sidebar_position: 5
+---
+
+# SSD Data Tiering in Dragonfly
+
+Dragonfly v1.21.0 introduces a powerful feature: SSD data tiering. This innovative approach optimizes
+storage utilization by leveraging SSD/NVMe disks as a secondary storage tier that complements
+your existing RAM. By intelligently offloading specific data to faster disk storage,
+Dragonfly can significantly reduce physical memory usage, potentially achieving a 2-5x improvement.
+
+## How It Works
+
+Dragonfly's data tiering focuses on string values exceeding 64 characters in length.
+When enabled, these longer strings are intelligently offloaded to the SSD tier,
+while shorter strings and other data types remain in memory. The main hashtable index is also kept
+in memory, which ensures high performant lookup operations.
+The offloaded data is still accessible and seamlessly loaded back into RAM when needed for reads.
```

Comment:
```suggestion
The offloaded data is still accessible and seamlessly loaded back into RAM when needed.
```
It's needed for both reads and writes, right? As such, I'd drop this and just say "when needed"

Reply:
it's not needed for writes. we do not fetch for writes.

Reply:
idk, it's still confusing. I imagine that for `APPEND`, which to me sounds like a "write" operation, we do read it. no?

Reply:
yes, for read-modify-write operations we only read it. I will change.